src/conv/k-g/k_parser.cpp

```diff
@@ -258,17 +286,51 @@ bool parse_k
 				if ((command.size() > 2) && (command[2] == "PULLEY")) {
 				    state = KState::Element_Beam_Pulley;
 
-				    elementOptions.insert(elementOptions.end(), command.begin() + 3, command.end());
+				    //elementOptions.insert(elementOptions.end(), command.begin() + 3, command.end());//ElEMENT_BEAM_PULLEY doesn't have options 
 				}
 				else if ((command.size() > 2) && (command[2] == "SOURCE")) {
 				    state = KState::Element_Beam_Source;
 
-				    elementOptions.insert(elementOptions.end(), command.begin() + 3, command.end());
+				    //elementOptions.insert(elementOptions.end(), command.begin() + 3, command.end());//ELEMENT_BEAM_SOURCE doesn't have options 
 				}
 				else {
 				    state = KState::Element_Beam;
 
 				    elementOptions.insert(elementOptions.end(), command.begin() + 2, command.end());
+
+				    if (elementOptions.size() > 0) {
+					for (size_t i_o = 0; i_o < elementOptions.size(); ++i_o) {
+					    if (elementOptions[i_o] == "ThICKNESS") {

```

Comment:
Shouldn't it be a capital H in ThICKNESS?

Reply:
yes, I was expecting some mistakes, I will check everything better if you think this is the right way to go